proto/kyve/multi_coin_rewards/v1beta1/types.proto

```
@@ -0,0 +1,57 @@
+syntax = "proto3";
+
+package kyve.multi_coin_rewards.v1beta1;
+
+import "amino/amino.proto";
```

Buf (1.47.2)

import "amino/amino.proto": file does not exist

(COMPILE)